tests/build/build.go

```diff
@@ -817,7 +817,8 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
 			}, timeout, interval).Should(BeTrue(), "timed out when waiting for the PipelineRun to fail")
 		})
 	})
-	Describe("Creating a component with a specific container image URL", Ordered, func() {
+	// Disabled due to PLNSRVCE-952
+	Describe("Creating a component with a specific container image URL", Pending, func() {
```

Comment:
```suggestion
		Describe("Creating a component with a specific container image URL", Ordered, Pending, func() {```
This is not working? I thought it should... And I think it would be better - this way it will be hard to forget the `Ordered` operator once enabling the test back.

Reply:
updated, disabled only two tests instead all of them